Fwk/AppFwk/cafProjectDataModel/cafPdmCore/cafPdmChildArrayField.inl

```diff
@@ -278,14 +278,15 @@ std::vector<DataType*> caf::PdmChildArrayField<DataType*>::children() const
 ///
 //--------------------------------------------------------------------------------------------------
 template <typename DataType>
-void PdmChildArrayField<DataType*>::children( std::vector<PdmObjectHandle*>* objects )
+std::vector<PdmObjectHandle*> PdmChildArrayField<DataType*>::children()
 {
```

Comment:
Maybe add [[NO_DISCARD]] on this one? https://en.cppreference.com/w/cpp/types/is_pointer